README.md

```diff
@@ -1,149 +1,114 @@
+
+1. Install PostgreSQL by downloading installers or packages from https://www.postgresql.org/download/. Alternatively, Mac and Linux users can use homebrew as shown below.
+
+   ```bash      
+   # Install homebrew.
+   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+   # Install PostgreSQL.
+   brew install postgresql
+   # Start DBMS service.
+   brew services start postgresql
+   ```
+
+2. Set up the PostgreSQL role (account) and databases for development and test (`pg-litefarm` and `test_farm`, respectively). You will use the `psql` client program. Account setup details will vary by OS. If an installer asks you to choose a password for the `postgres` (superuser) account, use `postgres`.
+
+   - Linux. In a terminal, start the client with `sudo -u postgres psql`, then execute each of the following commands. (The last command terminates the client session.)
+       ```sql
+       ALTER ROLE postgres WITH PASSWORD 'postgres';
+       CREATE DATABASE "pg-litefarm";
+       CREATE DATABASE test_farm;
+       exit;       
+       ```
+       
+   - Windows. At the Start menu, type `psql` and the search results will show "SQL Shell (psql)". In the client, execute each of the following commands. (The last command terminates the client session.)
+   
+       ```sql
+       CREATE DATABASE "pg-litefarm";
+       CREATE DATABASE test_farm;
+       exit;       
+       ```
+   
+3. In a terminal, navigate to the root folder of the repository. Run `npx lerna bootstrap` to install the dependencies for both applications.
+   
+4. Edit the `packages/api/.env` file, setting the value of variable `DEV_DATABASE_HOST` to `localhost`
+
+5. In the terminal, navigate to the `packages/api` folder. 
+
+6. Execute `npm run migrate:dev:db` to run the [migrations](https://knexjs.org/#Migrations) that set up the PostgreSQL database. (If you run into issues here, you can try dropping and recreating the database.) 
+
+8. Run `npm start` to launch the api application. 
+
+9. In a different terminal, navigate to the `packages/webapp` folder and run `npm start` to launch the webapp application. Load it in your browser at http://localhost:3000.
+
+# Testing
+
+## webapp
+
+To run [ESLint](https://eslint.org/) checks execute `npm run lint`
+
+Since this is a mobile web application, webapp should be viewed in a mobile view in the browser.
+
+Automated testing for the front-end is done using [cypress](https://www.cypress.io/). 
+
+2. In a terminal, navigate to the `packages/webapp` folder.
+2. Execute `npm test` to launch the tests.
+
+## api
+
+To run [ESLint](https://eslint.org/) checks execute `npm run lint`
+
+The [chai.js](https://www.chaijs.com/) and [jest](https://jestjs.io/) libraries automate tests that run real database operations using a [JWT](https://jwt.io/introduction) obtained through Auth0. The tests use a dedicated database named `test_farm`, distinct from the `pg-litefarm` database that the app normally uses .
```

Comment:
JWT tokens are no longer obtained through Auth0, so it might be better to take that part out.

Reply:
:+1: